### DISCUSSION TITLE
Can you please add more payment methods other than Paypal and PayLater?

### DISCUSSION THREAD

**Discussion by Hssnurrahman** [he3-app/he3-feedback#346] [Q&A] [answered]
Please add more payment methods. I am from Pakistan and Paypal does not work here & I want to pay.

**Comment by zhenweiwang1990** [comment, marked as answer]
Working on it. We will support Stripe for any credit cards later.

**Reply by Hssnurrahman** [reply]
Okay, I am waiting for that.
